### DISCUSSION TITLE
Fork with KMZ export and FTP upload

### DISCUSSION THREAD

**Discussion by Romanito** [SAHorowitz/MSFS2020-PilotPathRecorder#5] [Show and tell]
Hi,

Thanks for your great app! 
I was looking for a flight tracker for MSFS and had a look at your program as well as FsHub.
I like your app for the first person KML replays, using SimConnect over FSUIPC and being open source.
However I do like FsHub's web sharing of flight tracks, so I started coding something similar for PilotPathRecorder.

I made a [fork](https://github.com/Romanito/MSFS2020-PilotPathRecorder) with the following features:
- Export flights to KMZ files (smaller than KML)
- Upload files to a server via FTP or SFTP
- Made the main window more responsive to resizing and to initial SimConnect connection attempt
- Changed the exported file name pattern:
  - Sortable date first
  - Flight route (first and last waypoints)
  - Aircraft name

Feel free to review the [changes](https://github.com/Romanito/MSFS2020-PilotPathRecorder/commit/2366af2b7b152e2828bbf17db67788036381b4e5) and pull them back to your project (or not).

I also made a small [web app](https://github.com/Romanito/FlightTrackViewer) to view the uploaded track files. This way I can share my tracks on a self-hosted website and keep a hand on my data.

Keep up the good work!

**Comment by noodnik2** [comment]
Very cool, Romanito!  I'm curious to learn of your use case, and to see your solution in action - it'd be cool if you would make a quick demo video.  I'm now "following" you, in any case, and am looking forward to seeing the evolution of your fork!
